include/RAJA/policy/cuda/policy.hpp

```diff
@@ -220,19 +220,19 @@ const int RAJA_CUDA_MAX_BLOCK_SIZE = 2048;
 template <typename T>
 __device__ inline T _atomicMin(T *address, T value)
 {
-  return atomicMin(address, value);
+  return ::atomicMin(address, value);
```

Comment:
I like this use of the scope operator to make it clear when something is in the global namespace 👍